utils/node_local_test.py

```diff
@@ -867,28 +867,24 @@ def il_cmd(dfuse, cmd, check_read=True, check_write=True, check_fstat=True):
     """
     my_env = get_base_env()
     prefix = 'dnt_dfuse_il_{}_'.format(get_inc_id())
-    log_file = tempfile.NamedTemporaryFile(prefix=prefix,
-                                           suffix='.log',
-                                           delete=False)
+    log_file = tempfile.NamedTemporaryFile(prefix=prefix, suffix='.log', delete=False)
```

Comment:
(pylint-consider-using-with) Consider using 'with' for resource-allocating operations